install/kubernetes/helm/istio/templates/NOTES.txt

```diff
@@ -1,6 +1,6 @@
-Thank you for installing {{ .Chart.Name }}.
+Thank you for installing {{ .Chart.Name | title }}.
```

Comment:
Is the `title` one of predefined values for helm chart?
Because I didn't see it at: https://helm.sh/docs/charts/#predefined-values.

Reply:
@morvencao , `title` is a sprig function that can be used for Go templates along with the pipeline capability. Here is a reference to sprig: http://masterminds.github.io/sprig/strings.html